keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -874,46 +890,34 @@ Think about adding additional work or introducing new steps in between
 
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
-Mutating API calls that opt-in to validation will be slower ([initial
-benchmarks](https://github.com/kubernetes/kubernetes/pull/104433#issuecomment-901398507)
-estimate ~20% slower 25-30% more memory consumption)
+Mutating API calls that opt-in to validation will be slower.
+Benchmarks of alpha changes indicate that performing validation results in
+~2-5% slower request latency and ~4-8% more memory usage for json and yaml
+requests.
 
-<!--
-###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+Given that the majority of high-frequency requests made by system
+components use protobuf, we expect a negligible increase in overall resource
+usage.
 
-Things to keep in mind include: additional in-memory state, additional
-non-trivial computations, excessive access to disks (including increased log
-volume), significant amount of data sent and/or received over network, etc.
-This through this both in small and large cases, again with respect to the
-[supported limits].
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
 
-[supported limits]: https://git.k8s.io/community//sig-scalability/configs-and-limits/thresholds.md
+Non-negligible increase to RAM of kube-apiserver when server-side validation is
+heavily utilized.
 
 ### Troubleshooting
 
-This section must be completed when targeting beta to a release.
-
-The Troubleshooting section currently serves the `Playbook` role. We may consider
-splitting it into a dedicated `Playbook` document (potentially with some monitoring
-details). For now, we leave it here.
-
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+Same as without the feature
+
 ###### What are other known failure modes?
 
-For each of them, fill in the following information by copying the below template:
-  - [Failure mode brief description]
-    - Detection: How can it be detected via metrics? Stated another way:
-      how can an operator troubleshoot without logging into a master or worker node?
-    - Mitigations: What can be done to stop the bleeding, especially for already
-      running user workloads?
-    - Diagnostics: What are the useful log messages and their required logging
-      levels that could help debug the issue?
-      Not required until feature graduated to beta.
-    - Testing: Are there any tests for failure mode? If not, describe why.
+N/A
 
 ###### What steps should be taken if SLOs are not being met to determine the problem?
--->
+
+Opt out of server-side validation if the performance impact of it is not
```

Comment:
this requires finding all clients.  Do you mean setting the featuregate to false?

Reply:
yep. changed